src/Service.Tests/Configuration/HotReload/ConfigurationHotReloadTests.cs

```diff
@@ -443,7 +444,77 @@ await ConfigurationHotReloadTests.WaitForConditionAsync(
     }
 
     /// <summary>
-    /// 
+    /// Creates a hot reload scenario in which the schema file is invalid which causes
+    /// hot reload to fail, then we check that the program is still able to work
+    /// properly by validating that the DAB engine is still using the same configuration file
+    /// from before the hot reload.
+    /// </summary>
+    [TestCategory(MSSQL_ENVIRONMENT)]
```

Comment:
add in the summary, what is the invalid change that you're introducing in both the tests. This will make it easier to understand the tests.